BDArmory/UI/BDArmorySetup.cs

```diff
@@ -3106,6 +3106,8 @@ void WindowSettings(int windowID)
                         BDArmorySettings.MUTATOR_ICONS = GUI.Toggle(SLeftRect(++line, 1f), BDArmorySettings.MUTATOR_ICONS, StringUtils.Localize("#LOC_BDArmory_Settings_MutatorIcons"));
                     }
                 }
+                if (CheatOptions.InfinitePropellant) BDArmorySettings.INFINITE_FUEL = true; //syncronize settings options to Alt-F12 toggle status if it's been activated there but not here
```

Comment:
This `if-else` is the same as simply `BDArmorySettings.INFINITE_FUEL = CheatOptions.InfinitePropellant`.
Similarly for `INFINITE_EC` below.

Reply:
Looksl ike i was working off of a not-up-to-date fork and we both implemented this; removing mine;

Reply:
I don't think we both implemented this. I was just pointing out that
```csharp
if (A) B = true;
else B = false;
```
is the same as the more concise
```csharp
B = A;
```

I'll add in `BDArmorySettings.INFINITE_FUEL = CheatOptions.InfinitePropellant` for syncing with the alt-f12 window. I did notice that the check box doesn't sync the other way (BDA -> alt-F12), but I had a dig into the KSP API and it looks like the only way to do this is via some complicated reflection, so I don't think it's worth bothering about.